app/src/main/java/com/geeksville/mesh/service/RadioNotConnectedException.kt

```diff
@@ -0,0 +1,4 @@
+package com.geeksville.mesh.service
+
+open class RadioNotConnectedException(message: String = "Not connected to radio") :
```

Comment:
This exception is used outside of the radio service code.  Keeping it in its original location...